tests/e2e/visual/contacts/contact-user-hierarchy-creation.wdio-spec.js

```diff
@@ -0,0 +1,114 @@
+
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const contactPage = require('@page-objects/default/contacts/contacts.wdio.page');
+const genericForm = require('@page-objects/default/enketo/generic-form.wdio.page');
+const commonEnketoPage = require('@page-objects/default/enketo/common-enketo.wdio.page');
+const searchPage = require('@page-objects/default/search/search.wdio.page');
+const usersAdminPage = require('@page-objects/default/users/user.wdio.page');
+
+const utils = require('@utils');
+const { generateScreenshot } = require('@utils/screenshots');
+
+describe('Creating and editing contacts and users', () => {
+  const healthFacilityName = 'Nairobi North Facility';
+
+  before(async () => {
+    await loginPage.cookieLogin();
+    await commonPage.goToPeople();
+  });
+
+  after(async () => {
+    await utils.revertDb([/^form:/], true);
+  });
+
+  it('should create health facility, chw area and chw '+
+    'chw supervisor and chw user', async () => {
+    //create health facility
+    await generateScreenshot('new-facility', 'select-new-facility');
+    await commonPage.clickFastActionFlat({ waitForList: false });
+    await commonEnketoPage.selectRadioButton('Set the Primary Contact', 'Skip this step');
+    await generateScreenshot('new-facility', 'skip-primary-contact');
+    await genericForm.nextPage();
+    await commonEnketoPage.setInputValue('Name', healthFacilityName);
+    await generateScreenshot('new-facility', 'enter-facility-name');
+    await genericForm.submitForm({ waitForPageLoaded: false });
+    await contactPage.waitForContactLoaded();
+    await commonPage.hideSnackbar();
+    await generateScreenshot('new-facility', 'created-facility');
+
+    //create chw area and chw
+    await commonPage.clickFastActionFAB({ waitForList: false });
+    await browser.pause(150);
```

Comment:
I think we should explain these pauses because it's not straight forward 
```suggestion
    await browser.pause(150); // Waiting for animation to avoid blurry screenshots
```